pkg/api/aim/response/experiment.go

```diff
@@ -2,7 +2,7 @@ package response
 
 // GetExperiment represents the response json fot the GetExperimnt endpoint.
 type GetExperiment struct {
-	ID           int32   `json:"id"`
+	ID           string  `json:"id"`
```

Comment:
this change according to what `AIM` UI actually expected, so now we are fully consistent.